packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -701,17 +701,16 @@ class BuildOwner {
   /// This mechanism prevents build functions from transitively requiring other
   /// build functions to run, potentially causing infinite loops.
   ///
-  /// If the building argument is true, then this is a build scope. Build scopes
-  /// cannot be nested.
+  /// If the building argument is true, then this is a build scope.
```

Comment:
Maybe s/this is a build scope/additional asserts are enabled to check invariants that should apply during building/ or something.